libs/website/forms/src/lib/hooked-form-input/hooked-form-input.tsx

```diff
@@ -0,0 +1,54 @@
+import { ReactElement } from 'react';
+import { Input, InputProps } from '../../../../ui-kit/src/lib/input/input';
+import { TextareaProps } from '../../../../ui-kit/src/lib/textarea/textarea';
+import { Path, UseFormRegister } from 'react-hook-form';
+
+export enum FormState {
+  idle,
+  pending,
+  success,
+  error,
+}
+
+export interface FormError {
+  message: string | undefined;
+}
+
+export interface FormFields {
+  name?: string;
+  email?: string;
+  message?: string;
+}
+
+export interface HookedFormProps
+  extends Omit<InputProps, 'error' | 'onChange'>,
+    Omit<TextareaProps, 'onChange'> {
```

Comment:
remove second extend and create separate interface for textarea in hooked-form-textarea

```suggestion
export interface HookedInputProps
  extends Omit<InputProps, 'error' | 'onChange'> {
```

Reply:
done